profiles/mpl_funcs.py

```diff
@@ -0,0 +1,128 @@
+from matplotlib.offsetbox import AnnotationBbox, OffsetImage
+from matplotlib.animation import FuncAnimation
+
+from main_window.competition.comp_app import MAP_MARKER
+from main_window.device_manager import DeviceType
+from main_window.data_entry_id import DataEntryIds
+from main_window.main_app import MainApp
+from textwrap import wrap
+
+from main_window.mplwidget import MplWidget
+
+label_to_dataID = {"Altitude": DataEntryIds.CALCULATED_ALTITUDE,
+               "MaxAltitude": None,
+               "State": DataEntryIds.STATE,
+               "Stage2State": DataEntryIds.STATE,
+               "Pressure": DataEntryIds.PRESSURE,
+               "Acceleration": [DataEntryIds.ACCELERATION_X, DataEntryIds.ACCELERATION_Y,
+                                DataEntryIds.ACCELERATION_Z]
+               }
+
+label_unit = {"Altitude": "m",
+               "MaxAltitude": "m",
+               "State": "",
+               "Stage2State": "",
+               "Pressure": "" ,
+               "Acceleration": "g",
+               }
```

Comment:
You may be able to include this information as properties in the `Label` class. That way these key strings aren't like "magic numbers" which need to be synchronized in multiple places.